src/lib/stream/tests/ftp.test.js

```diff
@@ -0,0 +1,136 @@
+import * as fs from 'fs';
+import FTP from 'ftp';
```

Comment:
yarn docker:lint:fix
yarn run v1.22.21
$ docker-compose run --rm backend yarn lint:fix && docker-compose run --rm frontend yarn lint:fix
[+] Creating 2/0
 ✔ Container head-start-ttadp-redis-1  Running                                                                                    0.0s 
 ✔ Container postgres_docker           Running                                                                                    0.0s 
yarn run v1.22.19
$ eslint --fix src

/app/src/lib/stream/tests/ftp.test.js
  2:17  error  Unable to resolve path to module 'ftp'  import/no-unresolved

Reply:
I don't understand